spec/lib/blacklight/search_builder_spec.rb

```diff
@@ -187,6 +187,16 @@
     end
   end
 
+  describe "#facet" do
+    it "should be nil if no value is set" do
+      expect(subject.facet).to be_nil
+    end
+
+    it "should set facet value" do
+      expect(subject.facet('format').facet).to eq 'format'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.